sailfish-k8s/sailfish/base/foundation/runner-autoscaler.yaml

```diff
@@ -56,7 +56,7 @@ spec:
               memory: "250M"
               cpu: "750m"
             limits:
-              memory: "1000M"
+              memory: '1'
```

Comment:
is this community best practice? I personally like the unit to be included

Reply:
I changed these because Argo kept "OutOfSync Looping" because kubernetes changes 1000M to a equivalent number without a unit

Reply:
I'll add units back and see if that works